src/VPolygon.jl

```diff
@@ -276,6 +277,123 @@ function ∈(x::AbstractVector{N}, P::VPolygon{N})::Bool where {N<:Real}
     return true
 end
 
+"""
+    rand(::Type{VPolygon}; [N]::Type{<:Real}=Float64, [dim]::Int=2,
+         [rng]::AbstractRNG=GLOBAL_RNG, [seed]::Union{Int, Nothing}=nothing
+        )::VPolygon{N}
+
+Create a random polygon in vertex representation.
+
+### Input
+
+- `VPolygon`     -- type for dispatch
+- `N`            -- (optional, default: `Float64`) numeric type
+- `dim`          -- (optional, default: 2) dimension
+- `rng`          -- (optional, default: `GLOBAL_RNG`) random number generator
+- `seed`         -- (optional, default: `nothing`) seed for reseeding
+- `num_vertices` -- (optional, default: `-1`) number of vertices of the
+                    polygon (see comment below)
+
+### Output
+
+A random polygon in vertex representation.
+
+### Algorithm
+
+We follow the idea [here](https://stackoverflow.com/a/47358689) based on
+[P. Valtr. Probability that n random points are in convex
+position](https://link.springer.com/article/10.1007%2FBF01271274).
+
+The number of vertices can be controlled with the argument `num_vertices`.
+For a negative value we choose a random number in the range `1:10`.
+"""
+function rand(::Type{VPolygon};
+              N::Type{<:Real}=Float64,
+              dim::Int=2,
+              rng::AbstractRNG=GLOBAL_RNG,
+              seed::Union{Int, Nothing}=nothing,
+              num_vertices::Int=-1
+             )::VPolygon{N}
+    @assert dim == 2 "a polygon must have dimension 2"
+    rng = reseed(rng, seed)
+    if num_vertices < 0
+        num_vertices = rand(1:10)
```

Comment:
i suggest to take at least `rand(3:10)`, to avoid some corner cases

Reply:
I was also thinking about that, but sometimes it is helpful to have these corner cases for testing (my main application for random sets is random testing). Otherwise you will systematically exclude these corner cases. You can control this number with `num_vertices` if you want to, so I think we can keep it like that.

Reply:
Thinking about it, my argument also holds in the other direction. So I will change it.